ui-library-quickstart-teams-interop-meeting-chat/src/App.tsx

```diff
@@ -78,13 +78,22 @@ export const CallWithChatExperience = (props: CallWithChatExampleProps): JSX.Ele
  * Entry point of your application.
  */
 function App(): JSX.Element {
+  /**
+   * By default, the `richTextEditorEnabled` is set to false,
+   * which means the plain text editor will be used for the SendBox component and the MessageThread component's edit function.
+   * Change this value to true to use the Rich Text Editor instead,
+   * which provides rich text formatting, table inserting etc.
+   */
+    const richTextEditorEnabled = false;
```

Comment:
It might be a good idea to move this constant to 
```
const ENDPOINT_URL = '<Azure Communication Resource Endpoint URL>';
const USER_ID = '<Azure Communication User ID>';
const TOKEN = '<Azure Communication Service Resource Token>';
const TEAMS_MEETING_LINK = '<Teams Meeting Link>';
```

Reply:
@vhuseinova-msft, it would be nice to have all the variables together but the `compositeOptions` is actually what's passed into the `CallWithChatExperience`.
Not sure whether by moving richTextEditorEnabled into the `CallWithChatExperience` would be confusing to the Contosos.

Reply:
Sorry, I don't mean to move it to `CallWithChatExperience` just to move `richTextEditorEnabled` to the same section where all other constants, that Contoso needs to set, are. So it will be easily discoverable
We already doing this for `DISPLAY_NAME`. It's created on top of the file with other constants but used as 1 of the props values of `CallWithChatExperience`